specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -156,6 +156,76 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/providers/Microsoft.Subscription/createSubscription": {
+      "post": {
+        "description": "Creates an Azure subscription on behalf of a customer",
+        "operationId": "SubscriptionFactory_CustomerCreateSubscription",
```

Comment:
@jetts Would it make more sense to have the operation ID as CustomerSubscription_Create? The class created will be CustomerSubscription and the operation Create